drgn_tools/scsi.py

```diff
@@ -40,13 +42,45 @@ def for_each_scsi_host(prog: Program) -> Iterator[Object]:
             yield container_of(dev, "struct Scsi_Host", "shost_dev")
 
 
+def host_module_name(shost: Object) -> str:
+    """
+    Fetch the module name associated with the scsi host.
+    returns: the module name string.
+    """
+    try:
+        name = shost.hostt.module.name.string_().decode()
+    except ValueError:
+        name = escape_ascii_string("unknonw")
+    return name
+
+
 def print_scsi_hosts(prog: Program) -> None:
     """
     Prints scsi host information
     """
-    output = [["HOST", "NAME"]]
-    for x in for_each_scsi_host(prog):
-        output.append([hex(x.value_()), f"host{x.host_no.value_()}"])
+    output = [
+        ["SCSI_HOST", "NAME", "DRIVER", "Busy", "Blocked", "Fail", "State"]
+    ]
+    for shost in for_each_scsi_host(prog):
+        """
+        UEK7 kernel version 5.15 doesn't have the host_busy as part of
+        struct Scsi_Host
```

Comment:
Can you list the upstream commit id who removed the "host_busy" in the comment?

Reply:
Agreed, and please format it as a comment, e.g:
```python
# Since 6eb045e092ef ("scsi: core: avoid host-wide host_busy counter for scsi_mq"),
# host_busy is no longer a member of struct Scsi_Host.
```
The triple quote style should be reserved for function docstrings.